src/components/views/room_settings/AliasSettings.js

```diff
@@ -145,6 +147,7 @@ module.exports = React.createClass({
         if (!alias || alias.length === 0) return; // ignore attempts to create blank aliases
 
         const localDomain = MatrixClientPeg.get().getDomain();
+        if (!alias.includes(':')) alias += ':' + localDomain;
```

Comment:
Do we want to do the same thing for the canonical alias?

Reply:
no, 'cos the canonical alias is set via dropdown, so you can't get it wrong.